Apigee/SmartDocs/Revision.php

```diff
@@ -401,12 +409,28 @@ public function delete($revisionId = null)
      */
     public function importSwagger($swaggerUrl)
     {
+        $combinedString = rawurlencode($this->apiId) . '/revisions?action=import&format=swagger';
```

Comment:
You should not be changing the methods that are already here - they should be left and marked as deprecated since the API changed.